contents/teams/replay/objectives.mdx

```diff
@@ -1,24 +1,18 @@
-### 🎯 Goal 1: Quality and Consolidation for Session Replay (web & mobile)
+### 🎯 Goal 1: People think of PostHog as a mobile solution
 
-Why? 
+* Mobile replay for React Native
+* Lots of content on how to use PostHog on Mobile
 
-We've done a lot of investigation and preparation work and we want to capitalize on that
+### 🎯 Goal 2: Error tracking in people's hands
 
-* Mobile replay out of beta for Android/iOS
-* Make capture and playback as complete, performant and easy as possible
-* Build the best UX for finding and playing recordings
+* the top 5 features implemented
+* real customers using it and giving us feedback
+* integrated with CDP for notifications
+* integrated with replay for investigation
 
-### 🎯 Goal 2: Error tracking MVP
+### Goal 3: Hire two more people into the team
 
-Why? PostHog is "The single platform to analyze, test, observe, and deploy new features". Having network timings, network payloads, and console errors in Session Replay is a great developer tool. We want to learn by doing to understand how we could capture and expose even more information here to help people dig into what is happening with their products
+* as a goal because hiring takes time
```

Comment:
Wondering if it should be part of the goals, we mentioned during the retro that it should not have been part of the goals because we can't control it anyway, James said the same thing during the planning.
I'm fine either way.

Reply:
last time round @charlescook-ph mentioned it should be in the goals for any teams hiring

i don't have a _strong_ feeling one way or another - it does use up my time but only maybe a day a month 

if we hire then it'll take at least a week to onboard, and use all of our time to support the new person(s) as they get going